packages/ui-kit/src/utils/markdown-fragment-to-react.d.ts

```diff
@@ -0,0 +1,10 @@
+import type { ComponentType } from 'react';
+import type { Plugin } from 'unified';
+
+declare const markdownFragmentToReact: (
+  markdownString: string,
+  customElements?: Record<string, ComponentType>,
+  customPlugin?: Plugin
+) => JSX.Element;
+
+export default markdownFragmentToReact;
```

Comment:
Manual declaration file, so that we avoid using `allowJs` in the `tsconfig.json`.